pkg/agent/flowexporter/exporter/exporter_test.go

```diff
@@ -0,0 +1,179 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exporter
+
+import (
+	"strings"
+	"testing"
+	"time"
+	"unicode"
+
+	"github.com/golang/mock/gomock"
+	"gotest.tools/assert"
```

Comment:
do we need another dependency for this? why not use `testify/assert`?

Reply:
No, removed it.